views/forms/select2/code.html.twig

```diff
@@ -7,8 +7,20 @@
 
     {{ docs.heading('h2', 'Dependencies') }}
 
-    <p>Select2 elements need the <code>PulsarFormComponent</code> to be included in your browserify configuration.</p>
+    <p>Pulsar uses its own <a href="https://github.com/jadu/selectWoo">fork of SelectWoo</a>, which contains additional accessibility enhancements over the <a href="https://github.com/woocommerce/selectWoo">original selectWoo</a> which in turn offers better accessibility support over the <a href="https://select2.org/">original select2</a>.</p>
 
+    <p>You can find more information about the Jadu branch of selectWoo, such as screen reader coverage and known bugs on the <a href="https://github.com/jadu/selectWoo">Jadu/selectWoo GitHub repo</a></p>
+
+    {{ docs.heading('h3', 'package.json') }}
+
+    <p>The following should be included in your product’s <code>package.json</code> file.</p>
+
+    <pre><code lang="javascript">"dependencies": {
+    ...
+    "select2": "github:jadu/selectWoo#jadu-1.1.3",
```

Comment:
Should this not be pointing to master rather than a branch?

Reply:
It's pointing to the latest release, jadu-1.1.3.